contracts/NFTStorefrontV2.cdc

```diff
@@ -149,7 +149,7 @@ pub contract NFTStorefrontV2 {
         pub let salePrice: UFix64
```

Comment:
```suggestion
        access(all) let salePrice: UFix64
```

I assume that you haven't been able to actually try deploying this yet to see if it checks, right?

Reply:
I was focussing on NFTStorefront before moving onto v2. v1 deploys and updated transactions/scripts work